BUILD

```diff
@@ -77,7 +77,31 @@ linux_packages(
     depends = [
         "openjdk-8-jre"
     ],
-    modes = {
+    permissions = {
+        "./var/log/grakn/": "0777",
+    },
+    symlinks = {
+        "./usr/local/bin/grakn": "/opt/grakn/core/grakn",
+        "./opt/grakn/core/logs": "/var/log/grakn/",
+    },
+)
+
+
+rpm_package(
+    name = "deploy-rpm",
+    package_name = "grakn-core-bin",
+    installation_dir = "/opt/grakn/core/",
+    version_file = "//:VERSION",
+    spec_file = "//dependencies/distribution/rpm:grakn-core-bin.spec",
+    empty_dirs = [
+        "./var/log/grakn/",
```

Comment:
let's be consistent with the relative path by removing the leading `./`: `var/log/grakn/"`

Reply:
@lolski fixed in https://github.com/graknlabs/grakn/pull/4777/commits/1456af49e17ee353ec93c7b05a844d2ff421ab7c